packs/age-of-ashes-bestiary/soulbound-ruin.json

```diff
@@ -359,8 +369,7 @@
             "value": [
                 "construct",
                 "evil",
-                "undead",
-                "unholy"
```

Comment:
The addition of unholy was in 44d6a29e9c18, and was likely intentional.

Reply:
![image](https://github.com/user-attachments/assets/ece93993-2c89-4ecf-9ed1-d112cb29f1f1)
From the book

Reply:
> The addition of unholy was in [44d6a29](https://github.com/foundryvtt/pf2e/commit/44d6a29e9c1895dd582943f2fe68bfc48a133b80), and was likely intentional.

It was likely wrongly given the evil trait and when it got migrated it got given the unholy trait.

Reply:
Evil undead were given the unholy trait intentionally